bin/cli.js

```diff
@@ -165,16 +170,14 @@ function invoke(env) {
       `-x [${filetypes.join('|')}]`,
       'Specify the stub extension (default js)'
     )
-    .action(function(name) {
-      var instance = initKnex(env, commander.opts());
-      var ext = (
-        argv.x ||
-        env.configuration.ext ||
-        env.configuration.split('.').pop()
```

Comment:
I have no idea why we were trying to parse knexfile param for extensions, it looks wrong. I'm open to restoring it if someone explains why it was there in the first place.

Reply:
But how could it possibly work if `env.configuration` is either path to knexfile.js or resolved configuration object?

Reply:
buggily 😛